defaults/main.yml

```diff
@@ -49,10 +49,12 @@ aws_region: us-east-1
 aws_availability_zone: us-east-1a
 
 # What AWS instance type to create
+# instance_type: r5a.large
+# image_id: ami-01ca03df4a6012157
 instance_type: r5a.large
-image_id: ami-01ca03df4a6012157
+image_id: ami-08c40ec9ead489470
 
-acas_data_volume_size: 50
+acas_data_volume_size: 82
```

Comment:
Why change this?  It should be overridable in your overrides yamls and this just provides the default.

Reply:
Just to be clear. I think it's ok to change this but 82 is a strange number for a default and seems more likely to but too large for a blank new system.

Reply:
I will remove the comment

Reply:
I'm reverting it and adding the change to deployment automation